src/ansys/dpf/core/server.py

```diff
@@ -388,3 +390,46 @@ def get_or_create_server(server: BaseServer) -> Union[BaseServer, None]:
     if server:
         return server
     return _global_server()
+
+
+def available_servers():
+    """Searches all available installed DPF servers in the current machine
+    Binds new functions to the server module which helps to choose the appropritae/suitable version
+
+    after this call, you can do the following:
+
+    Examples
+    --------
+
+    >>> from ansys.dpf import core as dpf
+    >>> #out = dpf.server.available_servers()
+
+    launch a server.
+    >>> #server = dpf.server.start_2024_2_server()
+
+
+    Parameters
+    ----------
+    server: BaseServer, None
+
+    Returns
+    -------
+    server: returns the map of available DPF servers.
```

Comment:
```suggestion
    server: map of available DPF servers.
```
Because this is the "Returns" section, we don't need the "Returns the" part in the description for ``server``.